client/js/render.js

```diff
@@ -83,7 +83,7 @@ function buildChatMessage(data) {
 		text.html(templates.actions[type](data.msg));
 	}
 
-	if ((type === "message" || type === "action") && chan.hasClass("channel")) {
+	if ((type === "message" || type === "action" || type === "notice") && chan.hasClass("channel")) {
```

Comment:
Notices are processed in same callback on the server as normal messages and actions. So this change was needed to match ordering correctly.